lib/matplotlib/textpath.py

```diff
@@ -358,11 +349,21 @@ def get_glyphs_tex(self, prop, s, glyph_map=None,
                     warnings.warn("No supported encoding in font (%s)." %
                                   font_bunch.filename)
 
+                # Character is a glyph and needs to be mapped to corresponding index
                 if charmap_name == "ADOBE_STANDARD" and font_bunch.encoding:
+                    use_glyph = True
                     enc0 = dviread.Encoding(font_bunch.encoding)
-                    enc = {i: self._adobe_standard_encoding.get(c, None)
-                           for i, c in enumerate(enc0.encoding)}
+
+                    # Make a list of each glyph by splitting the encoding
+                    enc0_list = []
```

Comment:
No, that'll make a 2D list, but we need 1D.

The encoding that is generated looks like this
`['Grave/Acute/Circumflex/Tilde/Dieresis/Hungarumlaut/Ring.....']`
Thus splitting at "/" gives us individual character names.
Not only that, each index actually corresponds to its character code (eg: enc0_list[142] => 'uni20A9' which is the Won character)
Therefore, line 363-364, enumerates the list with `i => character code` and `c => character name` and creates a dictionary `character code => font index`
Later, in the code the glyph is retrieved using this dictionary
`if enc:
    charcode = enc.get(glyph, None)`

Hope this makes sense :)